charts/spire/charts/spiffe-oidc-discovery-provider/templates/deployment.yaml

```diff
@@ -1,3 +1,6 @@
+{{- if and (.Values.enabled) (eq .Values.tlsSecret "") (not .Values.certManager.enabled) (not .Values.insecureScheme.enabled) }}
+{{-   fail "You must specify tlsSecret or certManager.enabled or insecureScheme.enabled" }}
+{{- end }}
```

Comment:
Placeholder until https://github.com/spiffe/helm-charts-hardened/pull/163 merges. Then built in spire support takes its spot.